bundle.yml

```
@@ -100,6 +105,14 @@ options :
         help : Enable Field API variants [ON|OFF]
         cmake : BUILD_field_api={{value}}
```

Here, we should then enable fiat alongside field_api

```suggestion
        cmake : BUILD_fiat={{value}} BUILD_field_api={{value}}
```